framework/src/modules/nft/endpoint.ts

```diff
@@ -28,13 +28,18 @@ import {
 import { NFTStore } from './stores/nft';
 import { ALL_SUPPORTED_NFTS_KEY, LENGTH_ADDRESS, LENGTH_NFT_ID } from './constants';
 import { UserStore } from './stores/user';
-import { NFTJSON } from './types';
+import { ModuleConfig, NFTJSON } from './types';
 import { SupportedNFTsStore } from './stores/supported_nfts';
 import { NFTMethod } from './method';
 
 export class NFTEndpoint extends BaseEndpoint {
+	private _config!: ModuleConfig;
 	private _nftMethod!: NFTMethod;
 
+	public init(config: ModuleConfig): void {
```

Comment:
Can this please be called within `NFTModule.init` method, something of sort`this.endpoint.init({ ownChainID })`.